vault/core.go

```diff
@@ -708,6 +708,12 @@ func (c *Core) sealInitCommon(req *logical.Request) (retErr error) {
 		return retErr
 	}
 
+	// Create an identifier for the request
+	var err error
+	req.ID, err = uuid.GenerateUUID()
```

Comment:
Sorry, this comment should actually be for `vault/request_handling.go` -- when I was scrolling back up I didn't notice I was at the wrong place :-)